res/values-ua/strings.xml

```diff
@@ -0,0 +1,114 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+	<string translatable="false" name="help_url">https://github.com/sspanak/tt9/blob/master/docs/user-manual.md</string>
+	<string name="app_name" translatable="false">Traditional T9</string>
+	<string name="app_name_short" translatable="false">TT9</string>
+	<string name="app_settings">Налаштування TT9 Settings</string>
+	<string name="completed">Виконано</string>
+	<string name="error">Помилка</string>
+	<string name="no_language">Немає мови</string>
+
+	<string name="error_unexpected">Сталася неочікувана помилка.</string>
+	<string name="failed_loading_language_definitions">Не вдалося завантажити всі визначення мов.</string>
+
+	<string name="add_word_add">Додати</string>
+	<string name="add_word_confirm">Додати слово \"%1$s\" в %2$s?</string>
+	<string name="add_word_no_selection">Перемістіть курсор у слово, щоб додати його.</string>
+	<string name="add_word_blank">Пусте слово не додано.</string>
+	<string name="add_word_exist">Слово \"%1$s\" вже існує у словнику.</string>
+	<string name="add_word_invalid_language" translatable="false">Cannot add a word when no language is selected.</string>
+	<string name="add_word_success">\"%1$s\" додано.</string>
+	<string name="add_word_title">Додати слово</string>
+
+	<string name="pref_category_about">Про програму</string>
+	<string name="pref_category_abc_mode">Режим АБВ</string>
+	<string name="pref_category_hacks">Сумістність</string>
+	<string name="pref_category_appearance">Зовнішній вигляд</string>
+	<string name="pref_category_debug_options" translatable="false">Debug Options</string>
+	<string name="pref_category_predictive_mode">Предективний режим</string>
+	<string name="pref_category_function_keys">Гарячі клавіші</string>
+	<string name="pref_category_keypad">Клавіатура</string>
+	<string name="pref_category_setup">Початкові налаштування</string>
+	<string name="pref_category_usage_stats" translatable="false">Usage Stats</string>
```

Comment:
This is is used only in development mode, there is no need to translate it. Under normal circumstances the regular users shouldn't care about it. This applies to all sentences marked as `translatable="false"`